templates/terraform/modules/environment/variables.tf

```diff
@@ -32,14 +32,13 @@ variable "eks_worker_ami" {
   description = "The (EKS-optimized) AMI for EKS worker instances"
 }
 
-# {{ if .Config.Infrastructure.AWS.Cognito }}
-variable "auth_namespace" {
-  default = "cognito_auth"
-}
-variable "auth_pool_name" {
+# {{ if .Config.Infrastructure.AWS.Cognito.Deploy }}
+variable "user_pool" {
+  default = "{{ .Config.Name }}"
```

Comment:
This shouldn't be a default, you should provide this value as a parameter in `terraform/environments/<env>/main.tf`

Reply:
oh yup that sounds good.

Reply:
Okay just pushed out the changes. So maybe a rule is that for variables that are same across all environments we can use default values otherwise it should not have a default but instead be defined in the environment specific input variables.